### DISCUSSION TITLE
How to convert the output of Paddlocr to a Pandas dataframe

### DISCUSSION THREAD

**Discussion by 88arvin** [PaddlePaddle/PaddleOCR#8929] [General]
Using PaddleOCR, I was able to retrieve the table content from the scanned PDF after it had been converted to a jpg. Now, how can I convert the PaddleOCR output to a Pandas dataframe.
And is there any way we can directly extract the tabular data from the scanned PDF?

**Comment by Marbyun** [comment]
hi bro are you already solving this?

**Comment by tzktz** [comment]
> Using PaddleOCR, I was able to retrieve the table content from the scanned PDF after it had been converted to a jpg. Now, how can I convert the PaddleOCR output to a Pandas dataframe. And is there any way we can directly extract the tabular data from the scanned PDF?

did u find solution ?
